Utilities/bootstrap

```diff
@@ -476,10 +476,10 @@ def create_bootstrap_files(sandbox_path, args):
         elif target.is_swift and not target.is_library:
             link_output_path = os.path.join(bin_dir, target.name)
 
-            link_command = [args.swiftc_path,
+            link_command = [pipes.quote(args.swiftc_path),
                             '-o', pipes.quote(link_output_path)]
             if args.sysroot:
-                link_command.extend(["-sdk", args.sysroot])
+                link_command.extend(["-sdk", pipes.quote(args.sysroot)])
```

Comment:
This doesn't feel right, something else must be wrong here... we should be passing each argument in this array literally.

I think the actual probably might be in how we pass this command to swift-build-tool, maybe we are using the old arguments form?